### DISCUSSION TITLE
Issues with DDC mute control

### DISCUSSION THREAD

**Discussion by Anywhere-Music-Player** [MonitorControl/MonitorControl#1704] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Press Mute key on keyboard and after that volume will not work, after unmute volume will be at 1% and never change.
Only restart app required to fix this. 
Happens randomly, but often. 

### Steps to reproduce

1. Press Mute
2. Press Mute
3. Press any change volume keys - or + 
4. Notice actual volume of monitor stays on 1% but macOS volume notifitions shown volume change, which doesn't reflect monitor volume.

### Expected behavior

Volume works after Mute/Unmute

### Environment Information (please complete the following information)

```markdown
- macOS version: 15.2 (24C101)
- Mac model: M1 MacBook Pro 2021
- MonitorControl version: 4.3.3
- Monitor(s): Gigabyte M28U
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Some displays don't support mute/unmute commands properly. You might want to disable Mute DDC command if it is enabled under Advanced Settings. Alternatively you can try BetterDisplay as it has more options regarding Mute control:

https://github.com/waydabber/BetterDisplay/issues/1248#issuecomment-1627048299

**Reply by Anywhere-Music-Player** [reply]
> Some displays don't support mute/unmute commands properly. You might want to disable Mute DDC command if it is enabled under Advanced Settings. Alternatively you can try BetterDisplay as it has more options regarding Mute control:
> 
> [waydabber/BetterDisplay#1248 (comment)](https://github.com/waydabber/BetterDisplay/issues/1248#issuecomment-1627048299)

I just opened Advanced options(never did that before) and it's disabled already, but it's not actually.. 

![Screenshot](https://github.com/user-attachments/assets/e2d1e568-803f-487d-88eb-fa3412c0f61a)


better display maybe better, but super duper overwhelmed app with crazy amount of settings options, I don't need that, deal breaker.
